ci/build/release-prep.sh

```diff
@@ -0,0 +1,119 @@
+# 3. Update the code coverage badge in the README
+# 4. Open a draft PR using the release_template.md and view in browser
+# If you want to perform a dry run of this script run DRY_RUN=1 yarn release:prep
+
+set -euo pipefail
+
+main() {
+  if [ "${DRY_RUN-}" = 1 ]; then
+    echo "Performing a dry run..."
+    CMD="echo"
+  else
+    CMD=''
+  fi
+
+  cd "$(dirname "$0")/../.."
+
+  # Check that $GITHUB_TOKEN is set
+  if [[ -z ${GITHUB_TOKEN-} ]]; then
+    echo "We couldn't find an environment variable under GITHUB_TOKEN."
+    echo "This is needed for our scripts that use hub."
+    echo -e "See docs regarding GITHUB_TOKEN here under 'GitHub OAuth authentication': https://hub.github.com/hub.1.html"
+    exit
+  fi
+
+  # Check that hub is installed
+  if ! command -v hub &>/dev/null; then
+    echo "hub could not be found."
+    echo "We use this with the release-github-draft.sh and release-github-assets.sh scripts."
+    echo -e "See docs here: https://github.com/github/hub#installation"
+    exit
+  fi
+
+  # Check that they have jq installed
+  if ! command -v jq &>/dev/null; then
+    echo "jq could not be found."
+    echo "We use this to parse the package.json and grab the current version of code-server."
+    echo -e "See docs here: https://stedolan.github.io/jq/download/"
+    exit
+  fi
+
+  # Check that they have rg installed
+  if ! command -v rg &>/dev/null; then
+    echo "rg could not be found."
+    echo "We use this when updating files across the codebase."
+    echo -e "See docs here: https://github.com/BurntSushi/ripgrep#installation"
+    exit
+  fi
+
+  # Check that they have sd installed
+  if ! command -v sd &>/dev/null; then
+    echo "sd could not be found."
+    echo "We use this when updating files across the codebase."
+    echo -e "See docs here: https://github.com/chmln/sd#installation"
+    exit
+  fi
+
+  # Check that they have node installed
+  if ! command -v node &>/dev/null; then
+    echo "node could not be found."
+    echo "That's surprising..."
+    echo "We use it in this script for getting the package.json version"
+    echo -e "See docs here: https://nodejs.org/en/download/"
+    exit
+  fi
+
+  # credit to jakwuh for this solution
+  # https://gist.github.com/DarrenN/8c6a5b969481725a4413#gistcomment-1971123
+  CODE_SERVER_CURRENT_VERSION=$(node -pe "require('./package.json').version")
+  # Ask which version we should update to
+  # In the future, we'll automate this and determine the latest version automatically
+  echo "Current version: ${CODE_SERVER_CURRENT_VERSION}"
+  # The $'\n' adds a line break. See: https://stackoverflow.com/a/39581815/3015595
+  read -r -p "What version of code-server do you want to update to?"$'\n' CODE_SERVER_VERSION_TO_UPDATE
+
+  echo -e "Great! We'll prep a PR for updating to $CODE_SERVER_VERSION_TO_UPDATE\n"
+  # I can't tell you why but
+  # when searching with rg, the version needs to in this format: '3\.7\.5'
```

Comment:
I think by default `rg` uses regular expressions so the `.` matches any character and needs to be escaped to match a literal `.`.

One way around this would be to use `--fixed-strings` which treats the input as a literal string instead of a regular expression.

Reply:
TIL! I'll add that in and then merge! 🎉

Reply:
Ooh, didn't know about fixed-strings!